src/reactviews/pages/ConnectionDialog/components/connectButton.component.tsx

```diff
@@ -29,7 +29,6 @@ export const ConnectButton = ({
                 connectionDialogContext.state.connectionStatus ===
                 ApiStatus.Loading
             }
-            shape="square"
```

Comment:
The vscode aesthetic is shifting to match that of the Fluent default controls with their gently-rounded corners, so I've swapped the buttons to match.